### PR TITLE
Network error

### PR DESCRIPTION
**Pull request by kamiranoff**

### CODE REVIEW

File: src/assets/animations/index.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ const animations = {
   success: require('./success.json'),
   thumbsUp: require('./thumbUp.json'),
   thumbsDown: require('./thumbsDown.json'),
+  dish: require('./dish.json'),
```
Comment (heron2014) [inline src/assets/animations/index.ts:11]: are you still using dish?

File: src/api/types.d.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+export interface CreateResponse {
+  data?: {
+    owner: DecksTable['owner'];
+    cards: {
+      id: CardsTable['id'];
+      question: CardsTable['question'];
+      answer: CardsTable['question'];
+      frontend_id: CardsTable['frontend_id'];
```
Comment (heron2014) [inline src/api/types.d.ts:32]: frontend_id should be camelcase?

File: src/common/GeneralAlert/index.tsx [hunk 3]
```diff
@@ -56,15 +64,20 @@ const GeneralAlert = forwardRef<GeneralAlertRef, Props>(({ text, onAnimationFini
...
-  const renderContent = (message: NotificationMessages) => {
-    if (message === NotificationMessages.ERROR) {
+  const renderContent = (m: NotificationMessages) => {
+    if (isError || isNetworkError) {
       return (
         <>
           <View style={styles.lottie}>
-            <LottieView source={animations.thumbsDown} autoPlay loop style={styles.errorIcon} />
+            <LottieView
+              source={isError ? animations.thumbsDown : animations.dish}
```
Comment (heron2014) [inline src/common/GeneralAlert/index.tsx:73]: are we using animation dish?

File: src/screens/Playground/index.tsx [hunk 4]
```diff
@@ -121,10 +127,11 @@ const Playground: FC<Props> = ({ route: { params }, navigation: { goBack, naviga
   };
   return (
     <Container style={styles.container}>
+      <GeneralAlert ref={alertRef} />
       <CloseButton onPress={handleGoBack} />
       <Title title={deckDetail.title} />
       <View style={styles.shareButtonContainer}>
-        <PrimaryButton buttonText="Share" onPress={handleShareDeck} />
+        <PrimaryButton buttonText='Share' onPress={handleShareDeck} />
```
Comment (heron2014) [inline src/screens/Playground/index.tsx:134]: single quotes sneaked in

File: src/assets/animations/index.ts (hunk 1 above)
Comment (kamiranoff) [reply src/assets/animations/index.ts:11]: yes it is used in the network alert. But we need to find a better animation

File: src/api/types.d.ts (hunk 2 above)
Comment (kamiranoff) [reply src/api/types.d.ts:32]: will change this.

File: src/common/GeneralAlert/index.tsx (hunk 3 above)
Comment (kamiranoff) [reply src/common/GeneralAlert/index.tsx:73]: yes. for network errors

**Review by heron2014 — APPROVED** [review]

**Pull request merged**